doc/development.md

```diff
@@ -104,7 +104,7 @@ When an experimental version of the Ignition config spec (e.g.: `2.3.0-experimen
 - Any configs with a `version` field set to the previously experimental version will no longer pass validation. For example, if `2.3.0-experimental` is being marked as stable, any configs written for `2.3.0-experimental` should have their version fields changed to `2.3.0`, for Ignition will no longer accept them.
 - A new experimental spec version will be created. For example, if `2.3.0-experimental` is being marked as stable, a new version of `2.4.0-experimental` will now be accepted, and start to accumulate new changes to the spec.
 - The new stable spec and the new experimental spec will be identical. The new experimental spec is a direct copy of the old experimental spec, and no new changes to the spec have been made yet, so initially the two specs will have the same fields and semantics.
-- The HTTP `user-agent` header that Ignition uses whenever fetching an object and the HTTP `accept` header that Ignition uses whenever fetching a config will be updated to advertise the new stable spec.
+- The HTTP `user-agent` header that Ignition uses whenever fetching an object and the HTTP `accept`, `ID`, `VERSION_ID` (if present), and `VARIANT_ID` (if present) headers that Ignition uses whenever fetching a config will be updated to advertise the new stable spec.
```

Comment:
Can you capitalize Accept? (I know it's not part of this PR)

Reply:
Sure, what about user-agent?

Reply:
fixed Accept